src/modules/45-dashboards/components/DashboardList/__tests__/DashboardList.test.tsx

```diff
@@ -8,14 +8,31 @@
 import React from 'react'
 import { act, fireEvent, render, RenderResult, screen } from '@testing-library/react'
 import { TestWrapper } from '@common/utils/testUtils'
-import type { DashboardModel } from 'services/custom-dashboards'
+import * as customDashboardServices from 'services/custom-dashboards'
+import { DashboardsContextProvider } from '@dashboards/pages/DashboardsContext'
 import { DashboardType } from '@dashboards/types/DashboardTypes.types'
+import { FolderType } from '@dashboards/constants/FolderType'
 import type { StringKeys } from 'framework/strings'
 import DashboardList, { DashboardListProps } from '../DashboardList'
 
 const testTitle = 'test title'
 
-const testDashboard: DashboardModel = {
+const mockFolderOne: customDashboardServices.FolderModel = {
+  id: '1',
+  name: 'testName',
+  title: 'testTitle',
+  type: FolderType.ACCOUNT,
+  child_count: 0,
+  created_at: '01/01/2022'
+}
+
+const mockGetFolderResponse: customDashboardServices.GetFoldersResponse = {
+  resource: [mockFolderOne],
+  items: 0,
+  pages: 0
```

Comment:
Should these be `1`s?
```suggestion
  items: 1,
  pages: 1
```